.github/ISSUE_TEMPLATE/92_tech_open-review.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "[Tech Chair -> Open Practices Chair] Open Materials Review"
+about: Review for Open Materials Chair to fill out
+title: "[REVIEW] Open Materials Review"
+labels: review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Open Practices Chair
+- If there is anything in the list below that you are unsure of, you can reply to this issue (after it has been created) and tag the responsible organizers. Organizers' Github IDs are listed on [People page](https://www.journalovi.org/people.html).
+- If anything in the below list is missing (e.g. required paper sections, supplemental materials, etc), tag the authors on this issue and ask them to add those materials.
+- Once authors have provided any missing material (or if no material is missing) and all requirements are met, close this issue.
+
+### Open Materials Checklist
+- [ ] Check that the paper license is CC-BY
+- [ ] Check that the “Research materials” section in the paper exists.
+- [ ] Check that all links to repositories and materials can be reached.
+- [ ] Check that all materials are in appropriate long-term repositories (e.g. OSF, Zenodo, etc). Non-long-term hosting solutions, such as Github or self-hosted websites, are acceptable only if they are also archived on a long term repository like OSF or Zenodo. One easy way to archive + DOI an existing Github repo is with Zenodo; see [here](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content)
+- [ ] Check that links to supplements and materials are listed in the following locations:
+   - [ ] abstract (according to the [structured abstract requirement](https://www.journalovi.org/author-guide.html#abstract))
+   - [ ] research material statements section of the paper.
+- [ ] Check that all other requirements under the [transparency section of the author guide](https://www.journalovi.org/author-guide.html#transparency-requirements) are fulfilled.
```

Comment:
If there are violations, what should the Open Practice Chair do? When should they use this issue to discuss with the authors, and when should they create a new issue to track them?

Note: The authors can click on any checkboxes to change the status. So a sneaky author might just do that without changing anything in the article. :D

Reply:
> If there are violations, what should the Open Practice Chair do? When should they use this issue to discuss with the authors, and when should they create a new issue to track them?

Instructions on what to do are at the top of this issue:

> If anything in the below list is missing (e.g. required paper sections, supplemental materials, etc), tag the authors on this issue and ask them to add those materials.

Reply:
Uh. Now you know the number of lines available in my working memory. :D 

OK.